src/components/App.tsx

```diff
@@ -213,6 +214,17 @@ export default class App extends PureComponent<Props, State> {
 		});
 	};
 
+	focus = (id: string): void => {
+		this.setState(previousState => {
+			const elementWithIdOrNext =
+				previousState.focusables.find(e => e?.id === id)?.id ||
```

Comment:
```suggestion
				previousState.focusables.find(event => event?.id === id)?.id ||
```

Reply:
Not an event here, it's a `Focusable`. Should I call is focusable?

Reply:
Ah, yes.